packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -246,9 +262,22 @@ const BaseChart = ({
             />
           </HotKeys>
         </Resizable>
-        <div className='base-chart-timestamp' style={timestampStyle}>
-          <span>{timestampStart}</span>
-          <span>{timestampEnd}</span>
+        <div className='base-chart-timestamp-container' style={timestampStyle}>
+          {isPropertiesRefreshing && (
```

Comment:
noticed that the spinner shifts over the whole timestamp
ideally there would be padding when the spinner is not there so that it does not shift any elements if it appears
 
<img width="527" alt="Screenshot 2024-02-29 at 10 29 48" src="https://github.com/awslabs/iot-app-kit/assets/28601414/0117c5f1-7075-4c8b-8496-b33f74448ab5">
<img width="499" alt="Screenshot 2024-02-29 at 10 30 27" src="https://github.com/awslabs/iot-app-kit/assets/28601414/5dfecbee-5f10-4c90-9330-cb33c22509bc">

Reply:
updated.